rs-e2e/src/main/java/gov/hhs/cdc/trustedintermediary/rse2e/AzureBlobFileFetcher.java

```diff
@@ -4,24 +4,30 @@
 import com.azure.storage.blob.BlobContainerClient;
 import com.azure.storage.blob.BlobContainerClientBuilder;
 import com.azure.storage.blob.models.BlobItem;
-import com.azure.storage.blob.models.BlobProperties;
+import com.azure.storage.blob.models.ListBlobsOptions;
 import java.time.LocalDate;
+import java.time.ZoneId;
 import java.time.ZoneOffset;
 import java.util.ArrayList;
 import java.util.List;
 
 /**
  * The AzureBlobFileFetcher class implements the {@link FileFetcher FileFetcher} interface and
- * fetches files from an Azure Blob Storage container.
+ * retrieves files from an Azure Blob Storage container.
  */
 public class AzureBlobFileFetcher implements FileFetcher {
 
-    private static final FileFetcher INSTANCE = new AzureBlobFileFetcher();
+    // We're using UTC for now, but we plan to change the timezone to be more realistic to the
```

Comment:
I think we should keep all times in UTC.  The working timezones can change and be unknown to future engineers